docs/resources/dataarts_architecture_data_standard_template.md

```diff
@@ -0,0 +1,129 @@
+---
+subcategory: "DataArts Studio"
+---
+
+# huaweicloud_dataarts_architecture_data_standard_template
+
+Manages DataArts Architecture data standard template resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "workspace_id" {}
+
+resource "huaweicloud_dataarts_architecture_data_standard_template" "test"{
+  workspace_id = var.workspace_id
+
+  optional_fields {
+    fd_name    = "dataLength"
+    required   = true
+    searchable = true
+  }
+
+  optional_fields {
+    fd_name    = "dqcRule"
+    required   = true
+    searchable = false
+  }
+
+  custom_fields {
+    fd_name         = "custom_field_test"
+    optional_values = "value1;value2;value3"
+    required        = true
+    searchable      = true
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `workspace_id` - (Required, String, ForceNew) Specifies the workspace ID of DataArts Architecture. Changing this
+  parameter will create a new resource.
+
+* `optional_fields` - (Optional, List) Specifies the optional fields of the data standard template to be activated.
+
+  Changing this parameter will create a new resource.
```

Comment:
This field is not `ForceNew`, so this line should be removed.
Check the other parts.

Reply:
done